components/ai_chat/resources/page/api/index.ts

```diff
@@ -12,7 +12,7 @@ import * as Mojom from '../../common/mojom'
 export type State = Mojom.ServiceState & {
   initialized: boolean
   isStandalone?: boolean
-  visibleConversations: Mojom.Conversation[]
+  conversationMetas: Mojom.Conversation[]
```

Comment:
nit: The name seems a little strange to me, but just a nit. Up to you, but we could just call this `conversations` (or `conversationsMetadata`?).